lib/utils/blockchain-txs.ts

```diff
@@ -68,16 +74,13 @@ export async function makeCommitment({
   durationInYears,
   ownerControlledFuses,
   authenticatedAddress,
-}: {
-  name: string;
-  secret: string;
-  data: string[];
-  reverseRecord: boolean;
-  resolverAddress: string;
-  durationInYears: bigint;
-  ownerControlledFuses: number;
-  authenticatedAddress: `0x${string}`;
-}) {
+  publicClient,
+}: MakeCommitmentParams) {
+  const chain = publicClient.chain;
+
+  const nameRegistrationContracts =
+    nameRegistrationSmartContracts[chain.id as SupportedNetwork];
```

Comment:
Here we are trusting our frontEnd to send a supported chain ID. 

I prefer we explicitly test if the chain ID informed is valid in this other part of the architecture, to remove dependency from piece-to-piece trustability